sdks/python/apache_beam/transforms/core.py

```diff
@@ -2235,7 +2235,20 @@ def expand(self, pcoll):
 
 class Windowing(object):
   def __init__(self, windowfn, triggerfn=None, accumulation_mode=None,
-               timestamp_combiner=None):
+               timestamp_combiner=None, allowed_lateness=0):
+    """Class representing the window strategy.
+
+    Args:
+      windowfn: Window assign function.
+      triggerfn: Trigger function.
+      accumulation_mode: a AccumulationMode, controls what to do with data
+        when a trigger fires multiple times.
+      timestamp_combiner: a TimestampCombiner, determines how output
+        timestamps of grouping operations are assigned.
+      allowed_lateness(int): Maximum delay in seconds after end of window
```

Comment:
Nit: I'd cut the (int) for consistency. Technically we should accept floats here as well, and possibly convert to Durations.

Reply:
yeah, thanks.